src/Cassandra/Data/Linq/CqlExpressionVisitor.cs

```diff
@@ -608,6 +608,30 @@ private bool EvaluateMethod(string name, MethodCallExpression node)
                         .Append(")");
                     return true;
                 }
+                case "StartsWith":
+                    Visit(node.Object);
+                    var startsWithArgument = node.Arguments[0];
+                    var startString = (string)Expression.Lambda(startsWithArgument).Compile().DynamicInvoke();
+                    var endString = startString + char.MaxValue;
```

Comment:
I think `endString`must be
```csharp
                                      //hex representation of U+10FFFF
startString + Encoding.UTF8.GetString(new byte[] {0xF4, 0x8F, 0xBF, 0xBF})
```
As the [largest UTF8 codepoint according to the spec](https://tools.ietf.org/html/rfc3629#section-3) is [10FFFF](http://www.fileformat.info/info/unicode/char/10ffff/index.htm) and not [FFFF] (http://www.fileformat.info/info/unicode/char/ffff/index.htm) (Char.MaxValue).